site/documentation/content/api/telemetry/index.md

```diff
@@ -39,9 +39,9 @@ All other combinations are not supported by Hono and may result in the terminati
 
 As indicated above, it is up to the discretion of the protocol adapter whether it wants to use *AT LEAST ONCE* or *AT MOST ONCE* delivery semantics.
 
-Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when uploading telemetry data.
+Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data.
```

Comment:
the devices indicate the delivery semantics when they upload the data. Please revert